client/src/advertisement/about/index.tsx

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import "./index.scss";
+
+export function About() {
+  React.useEffect(() => {
+    document.title = "Finding the Right Mix";
```

Comment:
Usually we have a postfix:

```suggestion
    document.title = "Finding the Right Mix | MDN";
```